themes/default/layouts/partials/header.html

```diff
@@ -51,6 +51,11 @@
                                         Case Studies
                                     </a>
                                 </li>
+                                <li>
+                                    <a href="{{ relref . "/cloud-engineering-summit" }}">
+                                        Cloud Engineering Summit
```

Comment:
```suggestion
                                        2021 Summit
```

not sure if this is any better, but thought this might make it a bit more clear. id also advocate that we make it the 2nd item in the list below the overview.

Reply:
Seems reasonable to me --- makes it more clear this is an upcoming event so let's go with it!